c++/test/TestPredicatePushdown.cc

```diff
@@ -281,6 +282,50 @@ namespace orc {
     for (uint64_t i = 0; i < 2000; ++i) {
       EXPECT_EQ(i + 3500 , batch1.data[i]);
     }
+    // 2nd batch of 1500 rows
+    EXPECT_EQ(true, rowReader->next(*readBatch));
+    // test previous row number
+    EXPECT_EQ(5500, rowReader->getRowNumber());
+    EXPECT_EQ(1500, readBatch->numElements);
+    for (uint64_t i = 0; i < 1500; ++i) {
+      EXPECT_EQ(i + 5500 , batch1.data[i]) << i;
+    }
+    // no more batches
+    EXPECT_EQ(false, rowReader->next(*readBatch));
+  }
+
+  void TestSelectedWithStripeStats2(Reader* reader) {
```

Comment:
Can we have a more meaningful name? Or, could you add a comment as a function description?